mdk/version.py

```diff
@@ -22,4 +22,4 @@
 http://github.com/FMCorz/mdk
 """
 
-__version__ = "1.6.4"
+__version__ = "1.6.5"
```

Comment:
Hi @junpataleta,
_1.6.5_ or _1.7.0_ ?

HTH,
Matteo

Reply:
Hi Matteo,

From an earlier chat with Fred:
> Jun Pataleta, [09.01.18 09:06]
Hey Fred, thanks for merging my commits! would you like me to create a version bump commit, as well? (I already did one hehe)
https://github.com/junpataleta/mdk/commit/0a811cb64938e7f6f4b92eeb8220e7769dcfc0d6

> Fred Massart, [09.01.18 18:02]
Hi Jun. Thanks. I feel like we can go up to 1.7.0 with your patch.

Cheers!

Reply:
AFAIK I should read:
```diff
-__version__ = "1.6.4"
+__version__ = "1.7.0"
```
Edit: but... it has been merged so I think I'm wrong ;).

HTH,
Matteo

Reply:
No, no, you were right: 820806cc17978652f3dcfe543cc2e0debb04c582

Reply:
Aha! Correct! Sorry I messed that up and thanks Fred for sorting that one out!